server/src/main/java/org/opensearch/indices/IndicesService.java

```diff
@@ -215,18 +215,56 @@ public class IndicesService extends AbstractLifecycleComponent
     private static final Logger logger = LogManager.getLogger(IndicesService.class);
 
     public static final String INDICES_SHARDS_CLOSED_TIMEOUT = "indices.shards_closed_timeout";
+    public static final String INDICES_REQUEST_CACHE_DISK_CLEAN_THRESHOLD_SETTING_KEY = "cluster.request_cache.disk.cleanup_threshold_percentage";
+    public static final String INDICES_REQUEST_CACHE_DISK_CLEAN_THRESHOLD_SETTING_DEFAULT_VALUE = "50%";
+
+    public static final String INDICES_REQUEST_CACHE_DISK_CLEAN_INTERVAL_SETTING_KEY = "indices.request_cache.disk.cleanup_interval";
+
+
     public static final Setting<TimeValue> INDICES_CACHE_CLEAN_INTERVAL_SETTING = Setting.positiveTimeSetting(
         "indices.cache.cleanup_interval",
         TimeValue.timeValueMinutes(1),
         Property.NodeScope
     );
+
+    public static final Setting<TimeValue> INDICES_REQUEST_CACHE_DISK_CLEAN_INTERVAL_SETTING = Setting.positiveTimeSetting(
+        INDICES_REQUEST_CACHE_DISK_CLEAN_INTERVAL_SETTING_KEY,
+        TimeValue.timeValueMinutes(1),
+        Property.Dynamic,
+        Property.NodeScope
+    );
+
     public static final Setting<Boolean> INDICES_ID_FIELD_DATA_ENABLED_SETTING = Setting.boolSetting(
         "indices.id_field_data.enabled",
         true,
         Property.Dynamic,
         Property.NodeScope
     );
 
+    public static final Setting<String> INDICES_REQUEST_CACHE_DISK_CLEAN_THRESHOLD_SETTING =
```

Comment:
Could instead use Setting.DoubleSetting() from 0 to 100 (or 0.0 to 1.0), which has built in min value and max value arguments?